pack/core/core_2026.json

```diff
@@ -154,6 +164,23 @@
         "position": 42,
         "quantity": 2
     },
+    {
+        "code": "12043",
+        "cost": 1,
+        "deck_limit": 2,
+        "faction_code": "seeker",
+        "illustrator": "JB Casacop",
+        "name": "Unbridled Knowledge",
+        "pack_code": "core_2026",
+        "position": 43,
+        "quantity": 2,
+        "skill_intellect": 2,
+	    "skill_wild": 1,
```

Comment:
indent

Reply:
Why do these look different in the email I got about this as opposed to the GitHub web viewer?